tools/importer-rest-api-specs/components/dataapigeneratoryaml/templater_package_definition.go

```diff
@@ -0,0 +1,66 @@
+package dataapigeneratoryaml
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/go-yaml/yaml"
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+)
+
+type Package struct {
+	Name       string   `yaml:"Name"`
+	Operations []string `yaml:"Operations"`
+	Constants  []string `yaml:"Constants"`
+	Models     []string `yaml:"Models"`
+}
+
+func codeForPackageDefinition(resourceName string, input models.AzureApiResource) ([]byte, error) {
+	operationNames := make([]string, 0)
+	for operation := range input.Operations {
+		operationNames = append(operationNames, operation)
+	}
+	sort.Strings(operationNames)
+
+	operations := make([]string, 0)
+
+	for _, operationName := range operationNames {
+		operations = append(operations, fmt.Sprintf("%sOperation", operationName))
+	}
+
+	constantNames := make([]string, 0)
+	for k := range input.Constants {
+		constantNames = append(constantNames, k)
+	}
+	sort.Strings(constantNames)
+
+	constants := make([]string, 0)
+	for _, constantName := range constantNames {
+		constants = append(constants, fmt.Sprintf("%sConstant", constantName))
+	}
```

Comment:
(same here) we don't need to suffix these anymore, that was a workaround for reflection/all of these being output into the same c# namespace:

```suggestion
```